scripts/windows/setup/IotEdgeSecurityDaemon.ps1

```diff
@@ -679,6 +688,12 @@ function Setup-Environment([string] $ContainerOs) {
             }
         }
     }
+
+    if ((-not $BypassArchCheck) -and ($env:PROCESSOR_ARCHITECTURE -eq 'ARM')) {
+        Write-HostRed 'Installing IoT Edge is currently not supported on Windows ARM32.'
```

Comment:
Suggest changing to - 'IoT Edge is currently not supported on Windows ARM32. Check https://aka.ms/iotedge-platsup for supported platform information'

Reply:
Added "See https://aka.ms/iotedge-platsup for more details." to match the message in the Windows version check.